### ISSUE TITLE
Email verification

### ISSUE THREAD

**Issue by mtbitcr** [BitcreditProtocol/E-Bill#204]
Create email verification mechanism throe Twilio SendGrid.

**Comment by tompro** [comment]
When verifying here who is the verifying entity? When we are talking about a P2P setting an email would have to be verified for each peer individually so this must be for some central peer registry or other trusted entity.

**Comment by djekmusic** [comment]
@tompro all right, usually such functionality is implemented on server side, which is trusted by all clients. However our server part is not created yet (looking for a Dev). Thats why for prototype we decided to implement client side validation only. Validation request is initiated by our client and email is sent with localhost confirmation callback. So it can be validated only when User clicks Confirm button at the same host where client is installed. (We realize that in such edition its not really a reliable validation)

**Comment by tompro** [comment]
Don't get me wrong, we could do P2P verification using a second (eg. the push channel) comms to send back the token but this would only be valid between two parties and need a third mechanism (eg. WOT) to populate to other parties.

**Comment by djekmusic** [comment]
Correct, thats base identity mechanism in our system, key exchange process happens between two Users only after that they become mutually trusted. The same procedure User needs to perform with all his contacts. But as I mentioned email validation its not an identity process, its not relevant and not trusted by other Users.
